.github/workflows/built-branch.yml

```diff
@@ -65,7 +65,12 @@ jobs:
           git config --global user.name "$GITHUB_ACTOR"
 
           rm -rf .gitignore docker_tag output.log .github
-          mv .deployignore .gitignore
+
```

Comment:
We should mirror this same logic in the `built-tag.yml` action too

Reply:
good point. done!